### PR TITLE
fix Aux timer action and add aux timer feedback and preset

### PR DESCRIPTION
**Pull request by alex-Arc**
- remove feedback ColorNegative as it is replaced by TimerZone (upgrade script exists)
- fix aux timer actions was using the old `extratimer` naming
- add aux timer feedback
   - Aux Timer Playback state
   - Aux Timer negative
- add aux timer presets
   - Current aux time
   - Start/Stop Aux Timer
   - Pause Aux Timer

### CODE REVIEW

**Comment by alex-Arc** [comment]
@cpvalente

File: companion/manifest.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
 	"name": "getontime-ontime",
 	"shortname": "ontime",
 	"description": "Companion module for ontime",
-	"version": "4.0.2",
+	"version": "4.2.0",
```
Comment (cpvalente) [inline companion/manifest.json:6]: should be 4.1.0

**Review by cpvalente — COMMENTED** [review]
Let's change the version and I will make a release

**Comment by alex-Arc** [comment]
@cpvalente 
> Let's change the version and I will make a release

Done 👍🏽

**Pull request merged**